server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -930,11 +925,15 @@ private List<PodSpec> installPods(File workingDir, Map<String, Object> jobEnvCon
                 specs.add(mainpod);
             }
             else {
-                String subspecname = podnameparts[1];
```

Comment:
A cocopoad can have hierarchies of dependencies, not only a parent and child relationship. Take this into account and add all subspecs.